yb-voyager/cmd/config.go

```diff
@@ -75,6 +75,14 @@ var allowedTargetConfigKeys = mapset.NewThreadUnsafeSet[string](
 	"db-schema", "ssl-cert", "ssl-mode", "ssl-key", "ssl-root-cert", "ssl-crl",
 )
 
+var allowedYugabytedControlPlaneConfigKeys = mapset.NewThreadUnsafeSet[string](
```

Comment:
this is going to be a breaking change then, right? (Not that it's a problem, just checking..)